torchvision/prototype/transforms/functional/_deprecated.py

```diff
@@ -0,0 +1,71 @@
+import warnings
+from typing import Any
+
+import PIL.Image
+import torch
+
+from torchvision.prototype import features
+
+from ._meta import convert_color_space, convert_color_space_image_pil
+
+
+# TODO: this was copied from `torchvision.prototype.transforms._utils`. Given that this is not related to pytree / the
+#  Transform object, we should probably move it to `functional._utils`.
+def is_simple_tensor(inpt: Any) -> bool:
+    return isinstance(inpt, torch.Tensor) and not isinstance(inpt, features._Feature)
+
+
+def to_grayscale(inpt: PIL.Image.Image, num_output_channels: int = 1) -> PIL.Image.Image:
+    if not isinstance(inpt, PIL.Image.Image):
+        raise TypeError("Input should be PIL Image")
+
+    if num_output_channels not in (1, 3):
+        raise ValueError("num_output_channels should be either 1 or 3")
+
+    call = ", num_output_channels=3" if num_output_channels == 3 else ""
+    replacement = "convert_color_space(..., color_space=features.ColorSpace.GRAY)"
+    if num_output_channels == 3:
+        replacement = f"convert_color_space({replacement}, color_space=features.ColorSpace.RGB)"
+    warnings.warn(
+        f"The function `to_grayscale(...{call})` is deprecated in will be removed in a future release. "
+        f"Instead, please use `{replacement}`.",
+    )
+
+    # We can't use `convert_color_space_image_pil` since it only supports a subset of the input color spaces that PIL
+    # supports, namely `features.ColorSpace`.
+    # TODO: Can we even deprecate in favor of `convert_color_space` in such a case? Should we tell the users to call
+    #  `image.convert("RGB")` on their own?
+    output = inpt.convert("RGB")
+
+    if num_output_channels == 3:
+        output = convert_color_space_image_pil(inpt, color_space=features.ColorSpace.RGB)
+    return output
+
+
+def rgb_to_grayscale(inpt: Any, num_output_channels: int = 1) -> Any:
```

Comment:
Same here, why not call `_F.rgb_to_grayscale`?

Reply:
Let's discuss this in https://github.com/pytorch/vision/pull/6474#discussion_r952554861.